ui-grid/ui-grid.d.ts

```diff
@@ -601,6 +601,13 @@ declare module uiGrid {
          */
         enableFiltering?: boolean;
         /**
+        * False by default. When enabled, this adds a settings icon in the top right of the grid, 
+        * which floats above the column header. The menu by default gives access to show/hide columns, 
+        * but can be customized to show additional actions.
+        * @default false
+        */
+        enableGridMenu?: boolean;
```

Comment:
@joeskeen @vvakame Please let me know when this fix will be merged. 

Just fyi...PR has been submitted to update the UI-grid API documentation.

https://github.com/angular-ui/ui-grid/pull/4626